app/src/main/java/com/github/mateo762/myapplication/home/HomeActivity.kt

```diff
@@ -21,6 +21,8 @@ class HomeActivity : BaseActivity() {
         super.onCreateDrawer()
         bottomNavView = findViewById(R.id.bottomNav)
         bottomNavView.setOnItemSelectedListener(bottomNavListener)
+
+        //Retrieve the habits here and send to fragments
```

Comment:
Is this a TODO like comment? You may sign it as TODO then

Reply:
Great thanks, will do so now!